Libraries/LibWeb/HTML/HTMLAnchorElement.h

```diff
@@ -41,9 +41,11 @@ class HTMLAnchorElement final
     virtual void initialize(JS::Realm&) override;
     virtual void visit_edges(Cell::Visitor&) override;
 
+    // ^DOM::EventTarget
     virtual bool has_activation_behavior() const override;
     virtual void activation_behavior(Web::DOM::Event const&) override;
-    virtual bool has_download_preference() const;
+
+    bool has_download_preference() const;
```

Comment:
Presumably this should be moved out of the way of the parade of overridden base functions entirely instead of just given a single newline.